### PR TITLE
Updated the timelines and publ status

### PR DESCRIPTION
**Pull request by iherman**
These changes should go online when we communicate to the public about the change of course...

### CODE REVIEW

File: PublStatus.md [hunk 1]
```diff
@@ -9,51 +9,50 @@ layout: default
...
-| Lightweight Packaging of Web Publications† | (none yet) | ([ed](https://w3c.github.io/pwpub/spec/ocf-lite.html))         | [pwpub](https://github.com/w3c/pwpub/)                |
-| AudioBooks WP Profile | [wd](https://www.w3.org/TR/audiobooks/) | [ed](https://w3c.github.io/audiobooks/)         | [audiobooks](https://github.com/w3c/audiobooks/)                |
-| Web Publications Use Cases and Requirements‡ | [note](https://www.w3.org/TR/pwp-ucr/) | [ed](https://w3c.github.io/dpub-pwp-ucr/)| [dpub-pwp-ucr/](https://github.com/w3c/dpub-pwp-ucr//)                |
+| Document | Latest official release | Editors’ Draft | Repository | Note |
+|----------|:-----------------------:|:--------------:|:----------:|:---:|
+| Publication Manifests | (none yet) | [ed](https://w3c.github.io/pub-manifest/)         | [pub-manifest](https://github.com/w3c/pub-manifest/)    | Rec-track |
+| AudioBooks WP Profile | [wd](https://www.w3.org/TR/audiobooks/) | [ed](https://w3c.github.io/audiobooks/)         | [audiobooks](https://github.com/w3c/audiobooks/)                | Rec-track |
```
Comment (mattgarrish) [inline PublStatus.md]: Should we still be referring to this as a "WP Profile" given the changes?

File: PublStatus.md [hunk 2]
```diff
@@ -9,51 +9,50 @@ layout: default
...
+| Lightweight Packaging of Web Publications | (none yet) | [ed](https://w3c.github.io/lpf/)         | [lpf](https://github.com/w3c/lpf/) | Not rec-track
```
Comment (mattgarrish) [inline PublStatus.md]: Same question about appropriateness of "of Web Publications" in this title.

File: PublStatus.md (hunk 1 above)
Comment (iherman) [reply PublStatus.md]: True. I have changed it.

File: PublStatus.md (hunk 2 above)
Comment (iherman) [reply PublStatus.md]: Again, true, and have changed it

**Comment by iherman** [comment]
Per the comments of @mattgarrish : we will have to be careful of renaming the audiobook and the lpf documents

**Review by wareid — APPROVED** [review]

**Pull request merged**